source/extensions/extensions_build_config.bzl

```diff
@@ -302,7 +302,7 @@ EXTENSIONS = {
     # RBAC matchers
     #
 
-    "envoy.rbac.matchers.upstream_ip":     "//source/extensions/filters/common/rbac/matchers:upstream_ip_lib",
+    "envoy.rbac.matchers.upstream_ip_port":     "//source/extensions/filters/common/rbac/matchers:upstream_ip_port_lib",
     "envoy.rbac.matchers.upstream_port":   "//source/extensions/filters/common/rbac/matchers:upstream_port_lib",
```

Comment:
I think this needs to get removed? Or is this staying to match on ports only? Would it be better to just have a single matcher and allow address to be optional and if so any address will match? So basically either port, address, or both need to be set?

Reply:
Thanks @mattklein123 .
I can see how having both could be confusing but I have kept the ip-port matcher to qualify IP with Port for matching IP & Port (if provided). Wanted to keep Port matcher separate so that a) be independent (no dependency on IP) b) Have IP as first class citizen of the IP-Port matcher an Port as an added qualifier. 

Please let me know if that makes sense.

Reply:
I don't really follow. You have port optional in one of them. Why can't address be optional? Logically then shouldn't you have an address only matcher? I would remove the dedicated port matcher and just make a single matcher that requires either to be set or both.

Reply:
I was hoping that we can avoid configurations where neither IP or Port are provided. 

Will change to combine the matchers 👍

Reply:
Just reject that case in code and I think that will turn out great, thank you.